tests/test_integration.py

```diff
@@ -84,7 +84,7 @@ def _wait_thread_flush(self):
         Helper that waits for the thread flush
         """
         self.tracer.writer._worker.stop()
-        self.tracer.writer._worker.join()
+        self.tracer.writer._worker.join(None)
```

Comment:
Is the `None` necessary?

Reply:
So the default is 2 seconds, but for the tests I think it's way safer to use `None`. If we keep the default of 2 seconds, then there's a little chance that some events are not processed by the writer. If we use `None`, then we're sure the writer thread has finished and all spans have been treated. :)

Reply:
Makes sense, thanks!